src/Catalyst.Core.Modules.Dfs/Controllers/V0/KeyDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Catalyst.Core.Modules.Dfs.Controllers.V0
+{
+    /// <summary>
+    ///  A key to some data.
+    /// </summary>
+    public class KeyDto
+    {
+        /// <summary>
+        ///   Typically a CID.
+        /// </summary>
+        public string Key;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)